pkg/controller/rest/presentproof/operation.go

```diff
@@ -180,9 +180,27 @@ func (c *Operation) AcceptProposePresentation(rw http.ResponseWriter, req *http.
 //    default: genericError
 //        200: presentProofAcceptPresentationResponse
 func (c *Operation) AcceptPresentation(rw http.ResponseWriter, req *http.Request) { // nolint: dupl
+	var buf bytes.Buffer
+
+	if req.Body != nil {
+		// nolint: errcheck
+		_, _ = io.Copy(&buf, req.Body)
+	}
+
+	if !isJSONArray(buf.Bytes()) {
+		rest.SendHTTPStatusError(rw,
```

Comment:
nit: I'm not sure we want to *force* the user to provide friendly names

Reply:
under the hood, it is not a required data. I added only for the rest API. This restriction can be removed easily.